doc/classes/BaseMaterial3D.xml

```diff
@@ -217,6 +217,9 @@
 		<member name="grow_amount" type="float" setter="set_grow" getter="get_grow" default="0.0">
 			Grows object vertices in the direction of their normals.
 		</member>
+		<member name="heightmap_corrected_z" type="bool" setter="set_heightmap_corrected_z" getter="is_heightmap_corrected_z_enabled" default="true">
+			If [code]true[/code] applies a correction on the parallax effect to better represent depth under certain view angles. However it might introduce undesirable artifacts on boundaries.
```

Comment:
```suggestion
			If [code]true[/code], applies a correction on the parallax effect to better represent depth under certain view angles. However, this may introduce undesirable artifacts on boundaries.
```